faf-commons-api/src/main/java/com/faforever/commons/api/dto/CoopFaction.java

```diff
@@ -0,0 +1,7 @@
+package com.faforever.commons.api.dto;
+
+public enum CoopFaction {
+
+  // Order is crucial
```

Comment:
I don't think the order here is crucial, as we don't need to map it to an int anywhere